themes/ccc/config.ts

```diff
@@ -70,8 +70,8 @@ const config: TThemeConfig = {
   documentCategories: ["All", "Laws", "Policies", "UNFCCC Submissions", "Litigation", "Climate Finance Projects", "Offshore Wind Reports"],
   defaultDocumentCategory: "All",
   features: {
-    familyConceptsSearch: true,
-    knowledgeGraph: true,
+    familyConceptsSearch: false,
```

Comment:
This should be true as it includes case categories etc

```suggestion
    familyConceptsSearch: true,
```

Reply:
ah yeh same, sorry will revert.